docs/source/index.rst

```diff
@@ -30,7 +30,6 @@ Pyro Documentation
    :maxdepth: 2
    :caption: Contributed Code:
 
-   contrib.autoguide
```

Comment:
someone who understands docs please check this stuff

Reply:
this is correct.